rdt-app/src/main/java/io/ona/rdt_app/util/Utils.java

```diff
@@ -73,6 +75,11 @@ public static Date convertDate(String dateStr, String format) throws ParseExcept
         return simpleDateFormat.parse(dateStr);
     }
 
+    public static int convertDpToPixels(Context context, float dp) {
+        DisplayMetrics displayMetrics = context.getResources().getDisplayMetrics();
+        return (int) (dp * displayMetrics.density);
+    }
+
```

Comment:
Why note use the native method 
TypedValue.applyDimension(
    TypedValue.COMPLEX_UNIT_DIP,
    dip,
     context.getResources().getDisplayMetrics()
);

Reply:
Done.